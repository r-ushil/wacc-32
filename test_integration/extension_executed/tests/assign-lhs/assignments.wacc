begin
  pair(int, int) x = newpair(1, 2);

  fst x = 42;

  println fst x;
  println snd x
end