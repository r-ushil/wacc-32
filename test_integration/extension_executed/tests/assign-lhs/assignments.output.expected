Compiler exit code: 0
--------- stdout ---------
42
2
--------------------------
Exit code: 0
